src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -416,10 +456,13 @@ private void EmitDynamicBlockNode(int block)
         {
             throw new NotImplementedException();
         }
-
+        
         private void EmitImportNode(AssociativeNode node)
         {
-            throw new NotImplementedException();
+            //doing absolutely nothing is actually
+            //enough to import binaries.
+            //TODO do other important things here!
+            //see: ProtoAssociative.CodeGen.EmitImportNode
```

Comment:
@mjkkirschner do you think it could be useful to implement this for a separate, initial compile pass, say `CompilePass.ImportAssemblies` to prepopulate class and function tables like we do today (see `CLRModuleType.ParseSystemType`)? We can then use this information to our benefit during code emission and/or at runtime.

Reply:
I will file a followup on this.